### PR TITLE
Passing tests

### PR DESCRIPTION
**Pull request by sfarmer1**

### CODE REVIEW

File: .github/workflows/activation.yml [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+      id: getManualLicenseFile
+      uses: webbertakken/unity-request-manual-activation-file@v1.1
+      with:
+        unityVersion: 2019.3.13f1
+# Upload artifact (Unity_v20XX.X.XXXX.alf)
+    - name: Expose as artifact
+      uses: actions/upload-artifact@v1
+      with:
+        name: ${{ steps.getManualLicenseFile.outputs.filePath }}
+        path: ${{ steps.getManualLicenseFile.outputs.filePath }}
```
Comment (GabLeRoux) [inline .github/workflows/activation.yml:19]: Activation is already part of the following example file:  
https://github.com/Unity-CI/unity-actions-example/blob/main/.github/workflows/main.yml#L12-L41

File: .gitignore [hunk 2]
```diff
@@ -2,6 +2,7 @@
 # Created by https://www.toptal.com/developers/gitignore/api/unity
 # Edit at https://www.toptal.com/developers/gitignore?templates=unity
 
+
```
Comment (GabLeRoux) [inline .gitignore:5]: I'd remove this

File: .github/workflows/activation.yml (hunk 1 above)
Comment (sfarmer1) [reply .github/workflows/activation.yml:19]: Sorry, I'm just testing some things out on my fork and accidentally created a PR to merge them back into the main project. My bad!

Comment (GabLeRoux) [reply .github/workflows/activation.yml:19]: ahah no problem 👍 If you want to contribute to the example project afterward, don't hesitate to do so :)

**Pull request closed without merging**